rules/framework.bzl

```diff
@@ -407,6 +408,19 @@ the framework as a dependency.""",
             default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
             doc = "Needed to allow this rule to have an incoming edge configuration transition.",
         ),
+        "platform_type": attr.string(
+            mandatory = False,
+            doc =
+                """Internal - currently rules_ios uses the dict `platforms`
```

Comment:
worth validating that either the value is not set or that its set to a platform in the `platforms` dict?

Reply:
We should have this covered here:
https://github.com/bazel-ios/rules_ios/pull/188/files#diff-627e46d92a0f2032c906c4cc6a941fa44d4a85e2f89e83f3700812f66781a9eaR68